packages/frontend/src/view/pages/user/components/UserAssetTable.tsx

```diff
@@ -55,15 +62,10 @@ export function UserAssetsTable(props: UserAssetsTableProps) {
                 <a href={`/proof/${entry.vaultOrPositionId}`}>(proof)</a>
               )}
             </span>,
-            props.isMine && props.type === 'SPOT' ? (
-              <LinkButton
-                className="w-full"
-                href={`/forced/new/spot/${entry.vaultOrPositionId}`}
-              >
+            props.isMine && (
+              <LinkButton className="w-full" href={forcedActionLink(entry)}>
```

Comment:
I am not sure whether determining whether its spot or perpetual by assetId is the way to go. Although if that’s the case maybe we could get rid of the type prop?

Reply:
You are right, we should use tradingMode for determining the action link